config/testgrids/kubernetes/sig-node/config.yaml

```diff
@@ -118,3 +118,10 @@ test_groups:
     - target_config: Tests name
     - target_config: Context
     name_format: '%s [%s]'
+- name: ci-kubernetes-node-kubelet-features
+  gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-node-kubelet-features
+  test_name_config:
+    name_elements:
+    - target_config: Tests name
+    - target_config: Context
+    name_format: '%s [%s]'
```

Comment:
and since we control this format it could be something like
`%s vm-image:%s`
OR
`%s running-in:%s`
which might be more intelligible at a glance

Reply:
looks legit! https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md#more-informative-test-names